src/ice_usb.c

```diff
@@ -157,54 +151,21 @@ uint16_t const *tud_descriptor_string_cb(uint8_t index, uint16_t langid)
     return utf16;
 }
 
-#ifdef ICE_USB_UART0_CDC
-
-static void ice_usb_cdc_to_uart0(uint8_t byte)
-{
-    if (uart_is_writable(uart0)) {
-        uart_putc(uart0, byte);
-    }
-}
-
-static void ice_usb_uart0_to_cdc(void)
+void tud_cdc_line_coding_cb(uint8_t itf, cdc_line_coding_t const *coding)
 {
-    while (uart_is_readable(uart0)) {
-        uint8_t byte = uart_getc(uart0);
-        tud_cdc_n_write_char(ICE_USB_UART0_CDC, byte);
-    }
-}
-
-#endif
+    // printf("%s: coding=%p baud=%lu\n", __func__, coding, coding->bit_rate);
 
-#ifdef ICE_USB_UART1_CDC
-
-static void ice_usb_cdc_to_uart1(uint8_t byte)
-{
-    if (uart_is_writable(uart1)) {
-        uart_putc(uart1, byte);
+    /* Mimick the Arduino reboot signal:
+     * https://arduino.github.io/arduino-cli/dev/platform-specification/#1200-bps-bootloader-reset */
+    if (coding->bit_rate == 1200) {
+        reset_usb_boot(0, 0);
+        assert(!"not reached");
```

Comment:
Curious about why you wanted to move it :)

Reply:
Not to be between `tud_cdc_rx_cb_table` and `tud_cdc_rx_cb`. Mentioned it in previous PR.